frontend/app/pages.js

```diff
@@ -1,22 +1,77 @@
-import {sendJSON} from './sendJSON';
+import {initialiseEventSource} from './sse';
+
 /**
  * This module provides functions which display a certain page to the user.
  */
 
+/**
+ * Creates the html for the waiting game.
+ *
+ * @param gameID The id of the game.
+ * @param {HTMLElement} rootElement The element to which the html is added.
+ * @param {String} playerListID The id the player list element should have.
+ * @param {String} startButtonID The id the start game button should have.
+ */
+function createWaitingGameHTML({
+    gameID,
+    rootElement,
+    playerListID,
+    startButtonID,
+}) {
+    while (rootElement.firstChild) {
+        rootElement.firstChild.remove();
+    }
+
+    const heading = document.createElement('h1');
+    heading.innerHTML = `You are in game ${gameID}`;
+    const playerList = document.createElement('section');
+    playerList.id = playerListID;
+    const startButton = document.createElement('button');
+    startButton.style.type = 'button';
+    startButton.innerHTML = 'Start Game';
+    startButton.id = startButtonID;
+    startButton.disabled = true;
+
+    rootElement.appendChild(heading);
+    rootElement.appendChild(playerList);
+    rootElement.appendChild(startButton);
+}
+
 /**
  * Displays the page for a game in a waiting state.
  *
  * @param gameID The ID of the game to display.
  */
 export default function waitingGame(gameID) {
```

Comment:
Is it okay to have 'default' for this function even though there's more than one function in this file now?